ArmRobot/Assets/Scripts/DomainRandomization.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DomainRandomization : MonoBehaviour
+{
+    public GameObject cube;
+    public GameObject robot;
+    public GameObject table;
+
+
+    public float scaleObject = 0.1f;
```

Comment:
Is this meant to be the scale of the cube? If so, why not just read it off the cube GameObject?

Reply:
no It will be the scale of each object

Reply:
Oh ok.